lib/server.js

```diff
@@ -1,73 +1,110 @@
-const express = require('express');
-const fs = require('fs');
-const quote = require('regexp-quote');
-const path = require('path');
-const manifest = require('../ssr/ssr-manifest.json');
+require("dotenv").config();
+const express = require("express");
+const fs = require("fs");
+const quote = require("regexp-quote");
+const path = require("path");
+const manifest = require("../ssr/ssr-manifest.json");
 const appPath = path.join(__dirname, "../ssr", manifest["app.js"]);
 const renderer = require(appPath).default;
 const pageSize = 20;
 const versionFile = `${__dirname}/../version.txt`;
-const version = fs.existsSync(versionFile) ? fs.readFileSync(versionFile, 'utf8') : '0';
+const version = fs.existsSync(versionFile)
+  ? fs.readFileSync(versionFile, "utf8")
+  : "0";
+const axios = require("axios");
 
 // Disabled for now because it causes confusion when we update the data
 // const cache = {};
 
-module.exports = async function({ plants, nurseries }) {
+module.exports = async function ({ plants, nurseries }) {
   const port = process.env.PORT || 3000;
   const app = express();
   const publicDir = `${__dirname}/../public`;
   if (!fs.existsSync(publicDir)) {
     fs.mkdirSync(publicDir);
   }
   app.use(express.static(publicDir));
-  app.use('/images', express.static(`${__dirname}/../images`));
-  app.get('/api/v1/plants', async (req, res) => {
+  app.use("/images", express.static(`${__dirname}/../images`));
+  app.use(express.json());
+
+  app.post("/get-vendors", async (req, res) => {
+    const baseUrl = process.env.PAC_API_BASE_URL;
+    const apiKey = process.env.PAC_API_KEY;
+    const url = `${baseUrl}/Plant/FindVendorsForPlantName?plantName=${req.body.plantName}&zipCode=${req.body.zipCode}&radius=${req.body.radius}&limit=5`;
```

Comment:
As previously mentioned, please use the built-in capabilities of axios to build query strings from parameters in a way that guarantees proper escaping. Also make sure the received parameters are strings.

Reply:
Suggestion taken.  Putting in with commit momentarily